docs/docs-components/DocSearch.js

```diff
@@ -17,8 +17,21 @@ function SearchBox({ popoverZIndex }: {| popoverZIndex?: CompositeZIndex |}) {
             placeholder="Search Gestalt"
             aria-label="Search Gestalt docs"
           />
-          <Box position="absolute" top left marginTop={3} marginStart={4}>
-            <Icon icon="search" accessibilityLabel="" size="16" color="default" />
+          <Box
```

Comment:
Took this directly from our SearchField cmp

Reply:
Can you leave a comment indicating that? So we'll know to update this if we ever need to update the SearchField markup